src/plot.jl

```diff
@@ -0,0 +1,33 @@
+@recipe function f(tt::Table; indices = [1,2])
+    if length(indices) == 2
+        x_name, y_name = propertynames(tt)[indices]
```

Comment:
For generic Tables.jl, this would be: `x_name, y_name = Tables.columnnames(tt)[indices]`